tests/features/common/kie-common.feature

```diff
@@ -124,4 +124,30 @@ Feature: RHPAM and RHDM common tests
       | AUTH_LDAP_LOGIN_MODULE | required  |
     Then file /opt/eap/standalone/configuration/standalone-openshift.xml should contain <login-module code="LdapExtended" flag="required">
 
+  Scenario: Check KIE_SERVER_JBPM_CLUSTER flag enabled
+    When container is started with env
+      | variable                        | value                |
+      | JGROUPS_PING_PROTOCOL           | kubernetes.KUBE_PING |
+      | KIE_SERVER_JBPM_CLUSTER         | true                 |
+    Then container log should contain KIE Server's cluster for Jbpm failover is enabled.
+    And XML file /opt/eap/standalone/configuration/standalone-openshift.xml should contain value 60000 on XPath //*[local-name()='cache-container'][@name='jbpm']/*[local-name()='transport']/@lock-timeout
```

Comment:
Seems that you've removed the other values and now are checking only 3 properties, can you explain why?

Reply:
Because are included in the xpath expressions

Reply:
> Because are included in the xpath expressions
what do you mean? can't the other properties be checked as well?

Reply:
Added: https://github.com/jboss-container-images/jboss-kie-modules/pull/434/files#diff-8c04b4ebd9507eec423b68af126b75ede5926fea663d2958c3c0d94ef9961b2dR531-R533